apps/dh/app-dh/project.json

```diff
@@ -17,8 +17,8 @@
         "allowedCommonJsDependencies": ["date-fns-tz"],
         "assets": [
           {
-            "glob": "favicon.ico",
-            "input": "libs/dh/shared/assets/src",
+            "glob": "**/*",
+            "input": "libs/dh/shared/assets/src/assets/favicon",
```

Comment:
question: Are the favicon files included twice in the final bundle? Once by this entry and once by the next one?

Reply:
No what's copied here is a favicon folder which includes the icons in the different sizes

Reply:
Good catch, should be fixed now :)